packages/core/admin/admin/src/translations/de.json

```
@@ -60,15 +74,31 @@
   "Roles & Permissions": "Rollen & Berechtigungen",
   "Roles.ListPage.notification.delete-all-not-allowed": "Manche Rollen konnten nicht gelöscht werden, da sie mit Benutzern verknüpft sind",
   "Roles.ListPage.notification.delete-not-allowed": "Eine Rolle, die mit einem Benutzer verknüpft ist, kann nicht gelöscht werden",
+  "Roles.RoleRow.select-all": "Wähle {name} für Mehrfach-Aktionen",
+  "Roles.RoleRow.user-count": "{number, plural, =0 {#  user} one {#  user} other {# users}}",
```

This can become "Benutzer", since it will always be the same, regardless of the number.

```suggestion
  "Roles.RoleRow.user-count": "Benutzer",
```